src/skmultiflow/rules/nominal_attribute_class_observer.py

```diff
@@ -0,0 +1,84 @@
+from skmultiflow.rules.attribute_expand_suggestion import AttributeExpandSuggestion
+from skmultiflow.trees.attribute_class_observer import AttributeClassObserver
+
+
+class NominalAttributeClassObserver(AttributeClassObserver):
```

Comment:
What is the difference between `NominalAttributeClassObserver` defined in the `trees` module and this one?

Why not extend it?

Reply:
Different functionality